### PR TITLE
Add NoOpScaleUpStatusProcessor

### PR DESCRIPTION
**Pull request by bskiba**
This will be useful for the purposes of testing.
Also fix naming of Process() parameter.

### CODE REVIEW

File: cluster-autoscaler/processors/status/scale_up_status_processor.go [hunk 1]
```diff
@@ -42,3 +42,10 @@ type ScaleUpStatusProcessor interface {
 func NewDefaultScaleUpStatusProcessor() ScaleUpStatusProcessor {
 	return &EventingScaleUpStatusProcessor{}
 }
+
+// NoOpScaleUpStatusProcessor is a ScaleUpStatusProcessor implementations useful for testing.
+type NoOpScaleUpStatusProcessor struct{}
+
+// Process processes the state of the cluster after a scale-up.
+func (p *NoOpScaleUpStatusProcessor) Process(context *context.AutoscalingContext, state *ScaleUpStatus) {
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/processors/status/scale_up_status_processor.go]: s/state/status

Comment (bskiba) [reply cluster-autoscaler/processors/status/scale_up_status_processor.go]: I somehow managed to exclude the fix I was intending to do... Done

**Pull request merged**